java/ql/src/semmle/code/java/frameworks/Kryo.qll

```diff
@@ -3,19 +3,60 @@
  */
 
 import java
+private import semmle.code.java.dataflow.DataFlow
+private import semmle.code.java.dataflow.FlowSteps
 
 /**
  * The type `com.esotericsoftware.kryo.Kryo`.
```

Comment:
Should the other package name be mentioned here as well, or should the package name could just be removed?

Reply:
For reference, the `kryo` and `kryo5` packages seem to contain exactly the same content, see https://github.com/EsotericSoftware/kryo/pull/652.